### ISSUE TITLE
Sitemap - Problem with alternate href for homepage 

### ISSUE THREAD

**Issue by jlachancekffein** [studioespresso/craft-seo-fields#32]
Example

```xml
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9">
   <url>
      <loc>http://mydomain.com/fr-ca/</loc>
      <lastmod>2021-05-11</lastmod>
      <changefreq>always</changefreq>
      <priority>1.0</priority>
      <xhtml:link rel="alternate" hreflang="en-CA" href="http://mydomain.com/en-ca/__home__" />
      <xhtml:link rel="alternate" hreflang="en-US" href="http://mydomain.com/en-us/__home__" />
   </url>
</urlset>
```

You can see that the href keep the __home__ pattern.

I don't know if is the best solution, but this pattern is defined with \craft\base\Element::HOMEPAGE_URI.

```php

// src/craftCMS/vendor/studioespresso/craft-seo-fields/src/services/SitemapService.php :: Line 219

str_replace(\craft\base\Element::HOMEPAGE_URI, '', $site['uri']);
```

---------------------------------------

**Versions**

Craft: 3.5.9
SEO Fields 1.2.2

**Comment by janhenckens** [comment]
Hey @jlachancekffein, I have this fixed in the last beta for the 2.0.0 release. But I should so fix this for 1.2.X. I'll try to get this in this week, will let you know when it's fixed!

**Comment by janhenckens** [comment]
Hey @jlachancekffein, I have a fix for this out in 1.2.4, could you update and give it a try?

**Comment by janhenckens** [comment]
2.0.0 is out and also includes this fix.
